thingsearch/service/src/main/java/org/eclipse/ditto/thingsearch/service/common/config/SearchPersistenceConfig.java

```diff
@@ -38,6 +39,14 @@ public interface SearchPersistenceConfig {
      */
     ReadConcern readConcern();
 
+    /**
+     * Returns the {@code ThrottlingConfig} for the search Index update.
+     *
+     * @return the Search Index ThrottlingConfig.
+     * @since 3.7.0
+     */
+    ThrottlingConfig getThrottlingConfig();
```

Comment:
I would name it more specific.  
Reasoning: We should make clear what is throttled.

It is about throttling after a policy update - so e.g. name it (also everywhere else, including the config) maybe: `getPolicyModificationCausedSearchIndexUpdateThrottling()`